cmd/editorconfig/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"path/filepath"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"gopkg.in/ini.v1"
+
+	editorconfig "github.com/editorconfig/editorconfig-core-go"
```

Comment:
You don't need to declare the name of the package, since `editorconfig` is the default

Reply:
I see, removed `editorconfig` and also changed path to use package from gopkg.in .
Thanks!
https://github.com/editorconfig/editorconfig-core-go/pull/4/commits/d113d7a31ef53fa0b968a73c8efa0f922f484dca